gap/pkg/OscarInterface/gap/QQBar.gi

```diff
@@ -0,0 +1,195 @@
+BindGlobal( "QQBarFieldElementFam",
+  NewFamily( "QQBarFieldElement", IsQQBarFieldElement ) );
+
+SetIsUFDFamily( QQBarFieldElementFam, true );
+
+DeclareRepresentation( "IsQQBarFieldElementRep",
+  IsJuliaWrapper and IsAttributeStoringRep );
+
+BindGlobal( "QQBarFieldType",
+  NewType( QQBarFieldElementFam,
+           IsQQBarFieldElement and IsQQBarFieldElementRep ) );
+
+BindGlobal( "QQBarField_Julia", Oscar.algebraic_closure( Oscar.QQ ) );
+
+# We cannot assign the Julia type at load time.
+QQBarFieldElementMatrixType:= fail;
```

Comment:
Not so important but out of curiosity, why doesn't this work:
```suggestion
QQBarFieldElementMatrixType:= Oscar_jl.eval(Julia.Meta.parse(Julia.String("Matrix{QQBarFieldElem}")));
```

To make this nicer, perhaps we should add a variant of `JuliaEvalString` which takes two arguments: a module and a string.

Reply:
Some tests fail, this seems to be the same problem that was addressed in #2333, and I would say the easiest solution for the current problem is the same as there: Bind the variable in question inside Oscar's Julia code, not in GAP code.